Tests/Translation/Extractor/TwigExtractorTest.php

```diff
@@ -7,11 +16,26 @@
 use PrestaShop\TranslationToolsBundle\Twig\Extension\TranslationExtension;
 use Symfony\Component\Translation\MessageCatalogue;
 use Twig\Environment;
+use Twig\Error\Error;
 use Twig\Loader\FilesystemLoader;
 
 class TwigExtractorTest extends TestCase
 {
-    public function testExtractWithDomain()
+    public function testExcludeDirectories(): void
+    {
+        $extractor = $this->buildExtractor();
+
+        $extractor->setExcludedDirectories([]);
+        $this->assertEmpty($extractor->getExcludedDirectories());
+
+        $extractor->setExcludedDirectories(['folder1', 'folder2']);
+        $this->assertSame(['folder1', 'folder2'], $extractor->getExcludedDirectories());
+    }
```

Comment:
Setters / getters usually don't need to be tested. You test behavior with the setter instead.

Reply:
added it just to reach coverage threshold :D